egs/callhome_diarization/v2/run.sh

```diff
@@ -356,3 +358,50 @@ if [ $stage -le 11 ]; then
   # Compare to 8.69% in ../v1/run.sh
   echo "Using the oracle number of speakers, DER: $der%"
 fi
+
+# Variational Bayes resegmentation using the code from Brno University of Technology
+# Please see https://speech.fit.vutbr.cz/software/vb-diarization-eigenvoice-and-hmm-priors 
+# for details
+if [ $stage -le 12 ]; then
+  utils/subset_data_dir.sh data/train 32000 data/train_32k
+  # Train the diagonal UBM.
+  sid/train_diag_ubm.sh --cmd "$train_cmd --mem 20G" \
+    --nj 40 --num-threads 8 --subsample 1 --delta-order 0 --apply-cmn false \
+    data/train_32k $num_components exp/diag_ubm_$num_components
+
+  # Train the i-vector extractor. The UBM is assumed to be diagonal.
+  diarization/train_ivector_extractor_diag.sh \
+    --cmd "$train_cmd --mem 35G" \
+    --ivector-dim $ivector_dim --num-iters 5 --apply-cmn false \
+    --num-threads 1 --num-processes 1 --nj 40 \
+    exp/diag_ubm_$num_components/final.dubm data/train \
+    exp/extractor_diag_c${num_components}_i${ivector_dim}
+
+  # Converts diagonal UBM and ivector extractor to numpy array format
+  diarization/convert_VB_model.sh --cmd "$train_cmd" exp/diag_ubm_$num_components/final.dubm \
+    exp/extractor_diag_c${num_components}_i${ivector_dim}/final.ie exp/VB 
+fi
+
+if [ $stage -le 13 ]; then
+  output_rttm_dir=exp/VB/rttm
+  mkdir -p $output_rttm_dir || exit 1;
+  cat $nnet_dir/xvectors_callhome1/plda_scores/rttm \
+    $nnet_dir/xvectors_callhome2/plda_scores/rttm > $output_rttm_dir/x_vector_rttm
+  init_rttm_file=$output_rttm_dir/x_vector_rttm
+
+  # VB resegmentation. In this script, I use the x-vector result to 
+  # initialize the VB system. You can also use i-vector result or random 
+  # initize the VB system.
+  diarization/VB_resegmentation.sh --nj 20 --cmd "$train_cmd --mem 10G" \
+    --initialize 1 \
+    data/callhome $init_rttm_file exp/VB exp/VB/diag_ubm.pkl exp/VB/ie.pkl || exit 1; 
```

Comment:
As I mentioned earlier, I think these pkl should be regular Kaldi objections, and the conversions should happen internally.